Userland/Libraries/LibWeb/HTML/Navigable.cpp

```diff
@@ -1143,13 +1143,12 @@ WebIDL::ExceptionOr<void> Navigable::populate_session_history_entry_document(
             // 2. Set saveExtraDocumentState to false.
             saveExtraDocumentState = false;
         }
-
```

Comment:
?

Reply:
The other "otherwise" `else if` blocks did not have a newline between them, so this is just for consistency